wgpu-types/src/lib.rs

```diff
@@ -964,6 +969,9 @@ impl InstanceFlags {
         if let Some(bit) = env("WGPU_DEBUG") {
             self.set(Self::DEBUG, bit);
         }
+        if let Some(bit) = env("WGPU_GPU_BASED_VALIDATION") {
+            self.set(Self::GPU_BASED_VALIDATION, bit);
+        }
```

Comment:
issue: uh

This already exists a few lines below. 😅

Reply:
wooops